config/locales/it.yml

```diff
@@ -155,25 +195,138 @@ it:
       spree/zone:
         description: Descrizione
         name: Nome
+        default_tax: Zona di Tassazione di Default
+      spree/adjustment:
+        adjustable: Variabile
+        amount: Quantità
```

Comment:
"Quantità" means quantity but I think in this case "amount" is used to specify the price, maybe is better to use "Prezzo".
@alepore @mtylty what do you think?